README.md

```diff
@@ -1 +1,49 @@
 # Conduit Connector Elasticsearch
+
+## General
+The Elasticsearch plugin is one of [Conduit](https://github.com/ConduitIO/conduit) builtin plugins.
```

Comment:
@hariso Should I remove `builtin` here as well? 😁

Reply:
Oh yes!